lib/vdsm/storage/lvm.py

```diff
@@ -592,28 +592,28 @@ def _reload_single_pv(self, pv_name):
 
             return updated_pvs[pv_name]
 
-    def _reloadpvs(self, pvName=None):
+    def _reloadpvs(self, pv_name=None):
```

Comment:
Note for future change - using pv_name for None, single pv name or list of names is 
horrible!

We need to get rid of this by always calling with a list, and call this pv_names.